plugins/dynamodb-positions-storage/build.gradle

```diff
@@ -0,0 +1,32 @@
+plugins {
+    id "java"
+}
+
+jar {
+    manifest {
+        attributes(
+                'Plugin-Id': "${project.name}",
+                'Plugin-Version': "${project.version}",
+        )
+    }
+
+    into('lib') {
+        from configurations.compile
+    }
+}
+
+dependencies {
+    compileOnly 'org.projectlombok:lombok'
+    compileOnly 'com.google.auto.service:auto-service'
+
+    compileOnly 'org.springframework.boot:spring-boot-starter'
+    compileOnly 'org.springframework.boot:spring-boot-starter-validation'
+    compileOnly project(":api")
+    compileOnly 'io.projectreactor:reactor-core'
+
+    compile ('com.amazonaws:aws-java-sdk-dynamodb:1.11.192') {
```

Comment:
why not the `1.11.321`?

Reply:
the same version as it used to be, don't want to do the dependencies update during this PR, might cause hidden issues